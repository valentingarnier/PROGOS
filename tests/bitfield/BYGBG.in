5 i
1 0
2 1
1 4
3 2
5 0
