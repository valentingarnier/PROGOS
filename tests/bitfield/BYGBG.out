What size (2, 3, 4, 5)? What strategy ([B]rute force, B[i]tfield, [K]nuth)? Y Y Y Y Y
Please score attempt (positions, colors): B B B B Y
Please score attempt (positions, colors): G B B Y G
Please score attempt (positions, colors): B G Y B G
Please score attempt (positions, colors): B Y G B G
Please score attempt (positions, colors): Found solution: B Y G B G
